source/fe/ast/parser.d

```diff
@@ -361,6 +369,16 @@ struct Parser
 				nextToken; // skip integer
 				attribute = make!BuiltinAttribNode(start, BuiltinAttribSubType.extern_syscall, syscallNumber);
 				break;
+			case CommonIds.id_module.index:
+				expectAndConsume(TokenType.COMMA, "@extern(module");
+				expect(TT.STRING_LITERAL, "@extern(module,");
+				string value = cast(string)context.getTokenString(tok.index);
+				//import std.algorithm.iteration : filter;
+				//uint syscallNumber = value.filter!(c => c != '_').to!uint;
+				nextToken; // skip lib name
+				//TODO: pass value as data
```

Comment:
You can store intern module name in `context.idMap`, and store `Identifier` inside `data`.